apps/__main__.py

```diff
@@ -0,0 +1,39 @@
+'''The morph-tool command line launcher'''
+import logging
+
+import click
+from neurom import load_neuron
+
+logging.basicConfig()
+logger = logging.getLogger('morph_tool')
+logger.setLevel(logging.INFO)
+
+
+@click.group()
+def cli():
+    '''The CLI entry point'''
+
+
+@cli.command(short_help='A simple neuron viewer')
```

Comment:
doesn't it pull the short help from the docstring?

Reply:
Yep